src/Voron/Data/CompactTrees/CompactTree.Debug.cs

```diff
@@ -117,12 +117,12 @@ public void Verify()
 
     private void VerifyNode(ref CursorState current)
     {
-        var dictionary = _llt.GetEncodingDictionary(current.Header->DictionaryId);
+        var dictionary = _llt.GetEncodingDictionary(_state.TreeDictionaryId);
         if (current.Header->NumberOfEntries == 0)
             return;
 
-        _ = GetEncodedEntry(current.Page, current.EntriesOffsets[0], out var lastEncodedKey, out var lastEncodedKeyLengthInBits, out var l);
-
+        var lastEncodedKey = GetEncodedKeySpan(ref current, 0, out var lastEncodedKeyLengthInBits, out var l);
+        
```

Comment:
Check the warning here. It's ok?

`Use of variable 'lastEncodedKey' in this context may expose referenced variables outside of their declaration scope`

Reply:
Fixed